controllers/shooter.py

```diff
@@ -140,35 +133,24 @@ def tracking(self) -> None:
         accel = math.hypot(
             self.imu.getWorldLinearAccelX(), self.imu.getWorldLinearAccelY()
         )
-        if self.indexer.has_cargo_in_chimney() and self.shooter.is_at_speed():
-            if self._reject_through_turret and self.turret.is_on_target(
+        if self.indexer.has_cargo_in_chimney():
+            if self.shooter.flywheel_error() < 2 and self._reject_through_turret and self.turret.is_on_target(
                 math.atan(math.radians(45))
             ):
                 self.next_state("firing")
 
             if (
                 self._wants_to_fire
+                or self.auto_shoot
```

Comment:
This will cause the shooter to always fire when it is asked to fire. Is that intentional?